### DISCUSSION TITLE
v4.0.0 with basin form ... "Require Valid reCAPTCHA response" basin form is not supported

### DISCUSSION THREAD

**Discussion by jeffhx** [kdevo/osprey-delight#22] [Q&A]

**Comment by kdevo** [comment]
Hello @jeffhx,

first of all: Please do not use my form for requesting private support for the theme multiple times, and [start a public discussion](https://github.com/kdevo/osprey-delight/discussions) instead. This way, everyone can benefit from the comments. 

Coming back to your question(s) you've sent me to my private inbox: The theme is in a stable and production state (provided that you're using the master branch), with multiple different users which have successfully set it up before, also with Basin contact form. I believe that the form is not working for you because you've not completely followed [the instructions in the README](https://github.com/kdevo/osprey-delight#contact-form). Please double-check and compare with the screenshot. 

Provided that the issue's title "Require Valid reCAPTCHA response" is your error message which comes from Basin, the configuration error seems clear to me: You seem to have made a tick here, where it's not supposed to be:

![image](https://user-images.githubusercontent.com/10863868/123132607-34e72300-d44f-11eb-90f9-b990ebe6be2a.png)

Please ensure that there is **no** tick inside the box. Hope it helps!

I've left a separate comment for your PR https://github.com/kdevo/osprey-delight/pull/20.

**Reply by kdevo** [reply]
Hi @jeffhx,

just wanted to ask if it helped! Do you have a working form now?

**Comment by jeffhx** [comment]
Hello Kai, I have not been able to do more config around it. I was using
the old osprey theme and was trying to migrate to the osprey-delight. Will
have some feedback once I have spent some more time with this migration.
